src/webui/www/private/preferences_content.html

```diff
@@ -817,6 +817,208 @@
     </fieldset>
 </div>
 
+<div id="AdvancedTab" class="PrefTab invisible">
+    <fieldset class="settings">
+        <legend>QBT_TR(libtorrent Section)QBT_TR[CONTEXT=OptionsDialog]&nbsp;(<a href="https://www.libtorrent.org/reference.html">QBT_TR(Open documentation)QBT_TR[CONTEXT=HttpServer]</a>)</legend>
+	<table>
```

Comment:
you incorrectly used tab here. please recheck PR.

Reply:
Sorry but I don't understand ... Any problem here?

Reply:
it seems you used 1 tab instead of 4 spaces for indention here.